src/features/pages/404-error/content/not-found.module.scss

```diff
@@ -0,0 +1,17 @@
+@use 'features/styles/theme/theme-mixins' as *;
+
+.not_found {
+    block-size: 80svh;
+    padding-top: 8rem;
```

Comment:
i think, we have a utility class for padding-top

Reply:
okay